### PR TITLE
change_border_color

### PR DESCRIPTION
**Pull request by reboot-ua**

### CODE REVIEW

File: src/components/Signup.vue [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
       <div class="form-group">
         <label>First Name</label>
-        <input type="text" class="form-control form-control-lg" v-model="firstName"/>
+        <input  type="text" class="form-control focused form-control-lg" v-model="firstName"/>
```
Comment (OlegSkydonenko) [inline src/components/Signup.vue]: looks like you've added an excessive class, also remove please remove space character

File: src/components/Signup.vue [hunk 2]
```diff
@@ -20,7 +20,7 @@
 
       <div class="form-group">
         <label>Phone Number</label>
-        <input type="email" class="form-control form-control-lg" v-model="phoneNumber"/>
+        <input type="text" class="form-control form-control-lg" v-model="phoneNumber"/>
```
Comment (OlegSkydonenko) [inline src/components/Signup.vue]: change type to "tel"

File: src/components/Signup.vue [hunk 3]
```diff
@@ -67,5 +67,12 @@ export default {
         })
     }
   }
+
 }
+
 </script>
+<style scoped>
+  .form-control:focus {
+    border-color: #212529  !important;
```
Comment (OlegSkydonenko) [inline src/components/Signup.vue]: please avoid using "!important"

File: src/components/Signup.vue [hunk 4]
```diff
@@ -67,5 +67,12 @@ export default {
         })
     }
   }
+
 }
+
```
Comment (OlegSkydonenko) [inline src/components/Signup.vue]: please remove an excessive blank lines

**Review by OlegSkydonenko — CHANGES_REQUESTED** [review]

**Review by asenyk — APPROVED** [review]

**Review by OlegSkydonenko — APPROVED** [review]

**Pull request merged**